### PR TITLE
Windows terminal quake v1.6

### PR DESCRIPTION
**Pull request by flyingpie**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124182)

### CODE REVIEW

**Comment by flyingpie** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @flyingpie,

`Microsoft.WindowsTerminal` appears to be a dependency:
image

File: manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: flyingpie.windows-terminal-quake
+PackageVersion: 1.6
+InstallerType: portable
+Commands:
+- windows-terminal-quake
```
Comment (stephengillie) [inline manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml:5]:
```suggestion
- windows-terminal-quake
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.WindowsTerminal
```

Comment (flyingpie) [reply manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml:5]: Thank you!

**Comment by stephengillie** [comment]
Hi @flyingpie,

There was a 2nd error, but this was cleared by opening Windows Terminal. Now, when running, there is this error:
image

**Comment by flyingpie** [comment]
@stephengillie Thank you! I'm gonna do some more extensive testing, are you on Win10 or 11?

**Comment by stephengillie** [comment]
We use Windows 10 by default, but can specify Windows 11 with the `MinimumOSVersion` field.

File: manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.WindowsTerminal
```
Comment (stephengillie) [inline manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml:8]:
```suggestion
    - PackageIdentifier: Microsoft.UI.XAML.2.8
    - PackageIdentifier: Microsoft.WindowsTerminal
```
The install is still erroring, and I think it's having an issue with this dependency of the Windows Terminal.

Comment (flyingpie) [reply manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml:8]: @stephengillie Thank you for your feedback!

Could you explain what the UI.XAML.2.8 dependency changes or solves?

**Comment by stephengillie** [comment]
* https://github.com/microsoft/winget-pkgs/issues/128252

**Comment by flyingpie** [comment]
> * [[Package Issue]: Microsoft.UI.Xaml.2.8 #128252](https://github.com/microsoft/winget-pkgs/issues/128252)

Although the app works along side Windows Terminal, people can also install Terminal through other means.

Could we go with a version for now that does not have Windows Terminal as a dependency?

There might be a version in the future that works with other terminals even, so it could be considered more "correct" anyway.

**Comment by stephengillie** [comment]
If I'm understanding correctly, this is normal program operation.
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**